src/Visible.js

```diff
@@ -55,6 +55,7 @@ class Visible extends Component {
 
 		this._targets = [];
 		this._timer = null;
+		this._supportObserver = !!window.IntersectionObserver;
 		this._supportElementsByClassName = (() => {
```

Comment:
How about static method?
This code is generated each time you create an instance.